utility.sh

```diff
@@ -116,6 +116,30 @@ get_resource_version_number() {
   eval echo "$""$UP"_VERSIONNUMBER
 }
 
+get_resource_version_key() {
+  if [ "$1" == "" ] || [ "$2" == "" ]; then
+    echo "Usage: shipctl get_resource_version_key RESOURCE_NAME KEY"
+    exit 99
+  fi
+  UP=$(get_resource_name "$1")
+  RESOURCE_META=$(get_resource_meta $UP)
+  if [ ! -d $RESOURCE_META ]; then
+    echo "IN directory not present for resource: $1"
+    exit 99
+  fi
+  RESOURCE_VERSION_FILE=$RESOURCE_META/version.json
+  if [ ! -f $RESOURCE_VERSION_FILE ]; then
+    echo "version.json not present for resource: $1"
+    exit 99
+  fi
+  if [ "$2" == "versionName" ] || [ "$2" == "versionId" ] || [ "$2" == "versionNumber" ]; then
```

Comment:
a better way to do this would be to add all the variables in an array and just check if `$2` is in that array. 
- this way you'd be able to keep adding values to the array without touching the condition check